benefits/core/templates/core/base.html

```diff
@@ -102,17 +102,36 @@
       {% endblock main-content %}
     </main>
 
-    <footer id="footer">
-      <div class="container">
+    <footer id="footer" class="navbar py-0">
+      <div class="container d-none d-lg-block">
         <ul class="footer-links m-0 p-0 list-unstyled d-lg-flex gap-lg-4">
           <li>
-            <a class="footer-link m-0 ms-5 ps-lg-0" href="{% url "core:help" %}">{% translate "Help" %}</a>
+            <a class="footer-link m-0 p-0" href="{% url "core:help" %}">{% translate "Help" %}</a>
```

Comment:
@angela-tran Basically.. the bug came from changing `ps-5` to `ms-5` but _not_ also changing the `ps-lg-0` to `ms-lg-0`, so it added this extra margin-left you can see on Desktop https://dev-benefits.calitp.org/ right now

Reply:
But it doesn't matter anymore cos this PR will not show this footer on Mobile anymore.